web/cypress/integration/route/create-edit-duplicate-delete-route.spec.js

```diff
@@ -101,7 +104,11 @@ context('Create and Delete Route', () => {
     cy.contains('Confirm').click();
 
     cy.contains('Next').click();
-    cy.get(selector.nodes_0_host).type(data.host2);
+    cy.get(selector.nodes_0_host).type('@');
```

Comment:
It would be better to define a data param

Reply:
okay, declaring it right away, thanks for review

Reply:
done